src/platforms/android/common/gradle.mdx

```diff
@@ -264,3 +311,23 @@ Check [Integrations](/platforms/android/configuration/integrations/) page for mo
 To learn more about the internals of auto-instrumentation, check out this [blog post](https://blog.sentry.io/2021/12/14/bytecode-transformations-the-android-gradle-plugin).
 
 </Note>
+
+## Auto installation
+
+The plugin offers automated installation feature of the Sentry Android SDK and [Fragment](/platforms/android/configuration/integrations/fragment/), [Timber](/platforms/android/configuration/integrations/timber/) and [OkHttp](/platforms/android/configuration/integrations/okhttp/) integrations. With that, you don't need to add any dependencies but just use the Sentry Gradle plugin as the feature is enabled by default from version 3.1.0.
```

Comment:
hmm, I think I wasn't clear here, I meant:
> you can just use the Sentry Gradle plugin and that's it, because the feature (auto-installation) is enabled by default.

Does this make more sense?